### ISSUE TITLE
Add grammar

### ISSUE THREAD

**Issue by bigPapiTech** [bigPapiTech/slimThim#8]

**Comment by CarCarGoes** [comment]
Hi @bigPapiTech, I'm looking to make my 1st contribution. Do you need assistance with grammar? Happy to help :)

**Comment by bigPapiTech** [comment]
> Hi @bigPapiTech, I'm looking to make my 1st contribution. Do you need assistance with grammar? Happy to help :)

Hi! I'm very happy that you found my project! I really would appreciate help with defining a simple grammar. Are you interested in the definition-part of the grammar?

**Comment by CarCarGoes** [comment]
Yes I am interested! Thank you for your reply! Please let me know how I can help :) 

Sent from Yahoo Mail on Android 
 
  On Sat, Feb 25, 2023 at 1:10, ***@***.***> wrote:   



Hi @bigPapiTech, I'm looking to make my 1st contribution. Do you need assistance with grammar? Happy to help :)


Hi! I'm very happy that you found my project! I really would appreciate help with defining a simple grammar. Are you interested in the definition-part of the grammar?

**Comment by bigPapiTech** [comment]
Please help me define a basic grammar, which later on can be implemented. 
I added some for basic operators, documentation/grammar.txt

I wish the language be able to 
- arithmetic Operators
- relational Operators
- logical Operators
- assign variables
- run loops
- if then else
